### PR TITLE
Added synchronization requirements to the use case descriptions

### PR DESCRIPTION
**Pull request by chrisn**
In response, to #22, this PR adds some synchronization timing requirements.

Review feedback welcome.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/me-media-timed-events/pull/56.html" title="Last updated on Apr 7, 2020, 9:58 AM UTC (b192e1d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/me-media-timed-events/56/4405cdc...b192e1d.html" title="Last updated on Apr 7, 2020, 9:58 AM UTC (b192e1d)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -281,6 +286,10 @@ <h3>Audio stream with titles and images</h3>
           messages to be displayed, with information about when the content
           should be displayed in relation to the <a>media timeline</a>.
         </p>
+        <p>
+          The visual information should be rendered within a hundred millseconds
```
Comment (tidoust) [inline index.html:290]: s/millseconds/milliseconds

File: index.html [hunk 2]
```diff
@@ -376,9 +385,16 @@ <h3>Synchronized map animations</h3>
           map, e.g., pan and zoom, and allows annotations to be added and
           removed, e.g., markers, at specified times during media playback.
           Control can also be overridden by the user with the usual interactive
-          features of the map at any time, e.g., zoom. Concrete examples are
-          provided by the <a href="http://webvmt.org/demos">tech demos</a> at
-          the WebVMT website.
+          features of the map at any time, e.g., zoom. The rendering of the map
+          animation and annotations should usually be to within a hundred
+          millseconds or so to maintain good synchronization with the video.
```
Comment (tidoust) [inline index.html:390]: s/millseconds/milliseconds

File: index.html [hunk 3]
```diff
@@ -391,6 +407,10 @@ <h3>Media analysis visualization</h3>
           bounding boxes). A web application then uses this timed metadata
           to overlay labels and annotations on the video using HTML and CSS.
         </p>
+        <p>
+          This use cases requires frame accurate synchronization of the content
```
Comment (tidoust) [inline index.html:411]: s/use cases/use case

Comment (tidoust) [reply index.html:411]: Not part of this update, and I cannot comment on parts that didn't change, but there seems to be an extra "the" five lines above this line: s/that describe the when the objects/that describe when the objects

File: index.html (hunk 1 above)
Comment (chrisn) [reply index.html]: Thank you! I have run the document through a spell checker, which picked up a few more mistakes.

File: index.html (hunk 2 above)
Comment (chrisn) [reply index.html]: :+1:

File: index.html (hunk 3 above)
Comment (chrisn) [reply index.html]: :+1:

**Comment by chrisn** [comment]
Thanks for the review, @tidoust, also @rjksmith (who sent feedback privately). I'll go ahead and merge.

**Pull request merged**